mappings/net/minecraft/entity/ai/goal/GoalSelector.mapping

```diff
@@ -24,6 +24,14 @@ CLASS net/minecraft/class_1355 net/minecraft/entity/ai/goal/GoalSelector
 	METHOD method_35114 setTimeInterval (I)V
 		ARG 1 timeInterval
 	METHOD method_35115 getGoals ()Ljava/util/Set;
+	METHOD method_38063 containsAnyOf (Lnet/minecraft/class_4135;Ljava/util/EnumSet;)Z
+		ARG 0 goal
+		ARG 1 controls
+	METHOD method_38064 canAllGoalsBeReplacedBy (Lnet/minecraft/class_4135;Ljava/util/Map;)Z
+		ARG 0 goal
+		ARG 1 goalsByControl
+	METHOD method_38849 tick (Z)V
```

Comment:
maybe `tickGoals`, as the other tick ticks the selector itself and maintains the validity of goals?

Reply:
This can be called independently (and they do) - not sure which one is better.

Reply:
I know, but this one only ticks its goals while the other one ticks the selector's goal maintenance (addition/removal) as well